File: Application/Features/Games/Common/GameRequest.cs
using Application.Features.Genres.Common;

namespace Application.Features.Games.Command
{
    public record GameRequest
        (
        string Title,
        string Description,
        GenreRequest Genre,
        DateTime? ReleaseDate,
        string Language,
        string GameStatus,
        string Developer,
        string? Engine,
        int PegiRating,
        List<string> Platforms,
        bool SupportsCrossPlay
        );
}

File: angular-movieFrontend/src/Data/Request/GameRequest.ts
export interface GameRequest {
    title: string;
    description: string;
    genre: {
        name: string;
    };
    releaseDate: string;
    language: string;
    gameStatus: string;
    developer: string;
    engine: string;
    pegiRating: number;
    platforms: string[];
    supportsCrossPlay: boolean;
}